lib/sendgrid/helpers/mail/category.rb

```diff
@@ -8,7 +8,15 @@ def initialize(name: nil)
       @name = name
     end
 
-    def to_json(*)
+    def name=(name)
+      @name = name
+    end
+
+    def name
+      @name
+    end
```

Comment:
we already have `attr_accessor :name`. Why do we need this to write explicitly ?

Reply:
Yeah my bad. There were some merge conflicts and I think this was a left over.